library/std/src/sys/pal/zkvm/mod.rs

```diff
@@ -73,9 +72,11 @@ pub fn abort_internal() -> ! {
 }
 
 pub fn hashmap_random_keys() -> (u64, u64) {
-    let mut buf = [0u32; 4];
-    unsafe {
-        abi::sys_rand(buf.as_mut_ptr(), 4);
-    };
-    ((buf[0] as u64) << 32 + buf[1] as u64, (buf[2] as u64) << 32 + buf[3] as u64)
+    // let mut buf = [0u32; 4];
+    // unsafe {
+    //     abi::sys_rand(buf.as_mut_ptr(), 4);
+    // };
+    // (buf[0] as u64) << 32 + buf[1] as u64, (buf[2] as u64) << 32 + buf[3] as u64)
+
+    (0, 0)
```

Comment:
Should we just
```suggestion
    unimplemented!("Unsupported");
```

Reply:
No we don't want to do this as it will break hashmaps. The randomness if apparently used to to seed the hasher which prevents some weird worse-case performance of the hashmap implementation. Longer term we should see if we can feed in the fiat-shamir randomness here.